src/test/java/com/iexec/sms/tee/session/palaemon/PalaemonSessionServiceTests.java

```diff
@@ -241,15 +247,16 @@ void shouldGetAppPalaemonTokens() throws Exception {
                     IexecEnvUtils.IEXEC_INPUT_FILE_NAME_PREFIX + "1", "file1",
                     IexecEnvUtils.IEXEC_INPUT_FILE_NAME_PREFIX + "2", "file2"));
         assertThat(tokens)
-                .containsEntry(IEXEC_APP_DEVELOPER_SECRET_0, APP_DEVELOPER_SECRET_VALUE)
-                .containsEntry(IEXEC_REQUESTER_SECRET_0, REQUESTER_SECRET_VALUE);
+                .containsEntry(IEXEC_APP_DEVELOPER_SECRET_0, APP_DEVELOPER_SECRET_VALUE);
```

Comment:
Updating `createSessionRequest()` to support multiple requester secrets would be more inclusive

Reply:
See 086d9a8

Reply:
Thanks :pray: